src/main/java/dev/blaauwendraad/masker/json/KeyContainsMasker.java

```diff
@@ -197,50 +193,12 @@ private void visitObject(MaskingState maskingState, @CheckForNull KeyMaskingConf
      * @param keyMaskingConfig the {@link KeyMaskingConfig} for the corresponding JSON key
```

Comment:
Would be nice to add to this method JavaDoc where the index will be after masking

Reply:
Good catch :+1: